### ISSUE TITLE
Formerly working install no longer starts

### ISSUE THREAD

**Issue by kbreit** [MoJo2600/pihole-kubernetes#107]
I had a working installation until this morning when I noticed DNS wasn't working. It appears the pod isn't taking the proper volumes and is erroring out. It's on the latest version of the Helm chart. Did something change in 1.8.23 or other recent versions that would make this incompatible? 

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] 01-resolver-resolv: applying...
[fix-attrs.d] 01-resolver-resolv: exited 0.
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 20-start.sh: executing...
 ::: Starting docker specific checks & setup for docker pihole/pihole
  [✓] Update local cache of available packages
  [i] Existing PHP installation detected : PHP version 7.3.19-1~deb10u1

  [i] Installing configs from /etc/.pihole...
  [i] Existing dnsmasq.conf found... it is not a Pi-hole file, leaving alone!
  [✓] Copying 01-pihole.conf to /etc/dnsmasq.d/01-pihole.conf
chown: cannot access '': No such file or directory
chmod: cannot access '': No such file or directory
chown: cannot access '/etc/pihole/dhcp.leases': No such file or directory
```

Values file is...
```
adlists:
  - https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/facebook/all

dnsmasq:
  upstreamServers:
   - server=/my.domain/192.168.15.42

persistentVolumeClaim:
  enabled: true
  storageClass: managed-nfs-storage-persistent

ingress:
  enabled: true
  tls:
    - secretName: pihole-www-tls
      hosts:
        - my.domain
  hosts:
    - my.domain
  path: "/"

serviceWeb:
  loadBalancerIP: 192.168.15.41
  annotations:
    metallb.universe.tf/allow-shared-ip: pihole-svc
  type: LoadBalancer

serviceDns:
  loadBalancerIP: 192.168.15.41
  annotations:
    metallb.universe.tf/allow-shared-ip: pihole-svc
  type: LoadBalancer

DNS1: 1.1.1.2
DNS2: 1.0.0.2

admin:
  existingSecret: pihole-web-password
  passwordKey: WEBPASSWORD
```

**Comment by MoJo2600** [comment]
I seems to have something to do with the volumes. Without the volume definition it is working for me. The version was updated from pihole 5.1 to 5.2 and then 5.2.1. But I'm currently not aware of a change to the persistence settings in pihole.

**Comment by MoJo2600** [comment]
Maybe it is a setting on the NFS side. A issue with NFS has been solved in #39 - may this helps? I can't assist on NFS issues, i do not use NFS.

**Comment by kbreit** [comment]
There was a storage issue causing this problem. Closing the case
